html/partials/compare/unmanaged_files.html.haml

```diff
@@ -0,0 +1,59 @@
+- if @diff["unmanaged_files"]
+  %a.scope_anchor{ id: "unmanaged_files" }
+  #unmanaged_files_container.scope
+    .row
+      .col-xs-1
+        %a{ href: "#unmanaged_files" }
+          %img.over.scope_logo_big{ :src => "/assets/logo-unmanaged-files.png",
+            "data-toggle" => "popover",
+            :title => "Unmanaged Files",
+            "data-content" => "#{scope_help('unmanaged_files')}" }
+        %span.toggle{ title: "Collapse/Expand" }
+      .col-xs-11
+        %h2
+          Unmanaged Files
+          .scope-summary
+            - if @diff["unmanaged_files"].only_in1
+              %span.summary-part
+                #{@description_a.name}:
+                #{safe_length(@diff["unmanaged_files"].only_in1, :files)} files
+            - if @diff["unmanaged_files"].only_in2
+              %span.summary-part
+                #{@description_b.name}:
+                #{safe_length(@diff["unmanaged_files"].only_in2, :files)} files
+            - if @diff["unmanaged_files"].common
+              %span.summary-part
+                %a.show-common-elements{ href: "#unmanaged_files_both" }<
+                  both
+                = ": #{safe_length(@diff["unmanaged_files"].common, :files)} files"
+            - if diffable_unmanaged_files.length > 0
+              %span.summary-part
+                %a#diff-unmanaged-files{ "data-toggle" => "modal",
+                    "data-target" => "#diff-unmanaged-files" }
+                  Diff files
+    .row.scope_content.collapse.in
+      .row
+        .col-xs-1
+        .col-xs-5.table_container
+          - if safe_length(@diff["unmanaged_files"].only_in1, :files) > 0 |
```

Comment:
Don't use the `|` character to split up lines. Wrap on commas or extract code into helper.